### ISSUE TITLE
When clicking the "x" to close tabs, they should temporarily become equal width

### ISSUE THREAD

**Issue by nchlswtsn** [atom/tabs#243]
When rapidly closing several tabs, you have to click "x" and move your mouse a bit to locate the next "x" because the following tab's "x" is not in the same position due to increasing tab widths. This expansion of tab widths should occur only after a short delay of clicking "x" so that you can close all tabs before the adjustment occurs. 

Chrome uses this method of delayed tab width adjustment, for an example:

![chrometabs](https://cloud.githubusercontent.com/assets/8950740/11797365/4db6b018-a277-11e5-91a8-d6dc5835be8b.gif)

Note how the tabs don't change size until after you mouseleave the tab.

**Comment by mnquintana** [comment]
Thanks for the suggestion! This looks like a duplicate of https://github.com/atom/tabs/issues/129 – feel free to subscribe there for updates.
